templates/package.hamlet

```diff
@@ -17,7 +17,8 @@ $newline never
             <h1>
                #{pn} #
                <span .latest-version>
-                 #{displayedVersion} #
+                 <a href="https://hackage.haskell.org/package/#{pn}">
```

Comment:
It seems like a strange place to put a link. I'd rather add an explicit "Hackage" link (if we're going to have one at all), where we could also add a "Homepage" and "Repository" link if desired. (I'm on the fence about Hackage at all to be honest, sending people there may result in the standard problems of no-docs-available resurfacing accidentally.)